wordpress-mobile/WordPress-iOS
# Restore post status after app is killed

Fixes #9002 

To test:
 - Start the app
 - Create a new post
 - Add some media to it
 - Tap on publish async
 - Kill the app
 - Restart the app
 - See if post is listed and failed
 - See if you can tap to retry on the post
 - See if retry works.
